vars/javascript.groovy

```diff
@@ -88,6 +88,7 @@ def installDependencies (os, wantedNpmVersion, customModules, ignoreScripts = fa
   // }
   if (isWindows(os)) {
     bat 'npm config set msvs_version 2015 --global'
+    bat 'npm config set python c:\\python27\\python.exe --global'
```

Comment:
This will set python2.7 as the default python for all jobs. Are we sure that's gonna work?

Reply:
It worked in my case. I'd rather do this quick fix and see if anything else breaks. If you use Python 3, you'd normally expect to use the python3 binary and not just "python".